src/Illuminate/Hashing/ArgonHasher.php

```diff
@@ -60,13 +60,13 @@ public function __construct(array $options = [])
      */
     public function make($value, array $options = [])
     {
-        $hash = password_hash($value, $this->algorithm(), [
+        $hash = @password_hash($value, $this->algorithm(), [
```

Comment:
I'm not sure if we want the error suppression here?

Reply:
Yeh, we do want it, otherwise the code bellow doesn't run always. The idea here is that we wanna have a standard interface to our hasher on both PHP 7 and PHP 8. If it fails, they we should always get a laravel exception out the other end.